payment/config/config.go

```diff
@@ -0,0 +1,50 @@
+package config
```

Comment:
add `config.yml.example` as a template, in case someone else want to run this, so they can easily fill in the variables

Reply:
see user service